src/components/Chat.js

```diff
@@ -98,23 +99,13 @@ function Chat({
           "We detected that your message contains profane language. If you think this was a mistake, please let us know!"
         );
       } else {
-        let ref = await firebase
-          .database()
-          .ref(`users/${user.id}/canChat`)
-          .once("value");
-        let canChat = ref.val();
-        if (!canChat) {
-          ref = await firebase
-            .database()
-            .ref(`/userGames/${user.id}`)
-            .once("value");
-          canChat = ref.numChildren() >= MIN_GAMES;
-          firebase.database().ref(`users/${user.id}/canChat`).set(canChat);
-        }
+        const canChat = user.firstOnline
+          ? new Date().getTime() - user.firstOnline > NEW_ACCOUNT_TIME
```

Comment:
Not sure how you feel about using `getTime` as opposed to retrieving the server time.